app/services/planning_application_creation_service.rb

```diff
@@ -118,4 +119,8 @@ def result_params
   def payment_amount_in_pounds(amount)
     amount.to_f / 100
   end
+
+  def may_be_immune?(planning_application)
```

Comment:
It might be worth keeping the method names consistent so we're not having to remember which is 'maybe' and which is 'possibly'?

```suggestion
  def possibly_immune?(planning_application)
```